runtime.json

```diff
@@ -154,5 +158,27 @@
                 "value": ".*[cC]onnected to data broker.*"
             }
         ]
+    },
+    {
+        "id": "mockservice",
+        "interfaces": [],
+        "config": [
+            {
+                "key": "enabled",
+                "value": "${{ useVehicleMock }}"
+            },
+            {
+                "key": "image",
+                "value": "ghcr.io/boschglobal/kuksa.val.services/mock_service:latest"
```

Comment:
Please don't use latest to avoid incompatible changes. Use semantic version vX or vX.Y to reference on latest in particular MAJOR or MINOR version. If Kuksa doesn't support such - use full semantic version: vX.Y.Z

Reply:
The problem is that there is no tagged version of the mock service, yet. It will only get a tagged release in August in the official kuksa.val.services repository. For the time being this "latest" image will only be created when manually triggered as there are no CI workflows which run upon push to main.

The URL will anyway change once a proper tagged release is available hence a change is inevitable.

Reply:
Just create git issue as reminder to check in August...